plugins/modules/net_tools/nmcli.py

```diff
@@ -1288,6 +1336,27 @@
     autoconnect: true
     state: present
 
+- name: Create a VPN L2TP connection for ansible_user to connect on vpn.example.com authenticating with user 'brittany' and pre-shared key as 'Brittany123'
+  block:
+    - name: Encript pre-shared key
+      shell: echo "0s"$(base64 <<<'Brittany123' | rev | cut -c5- | rev)
```

Comment:
Please use an Ansible jinja2 expression to compute this instead of a `ansible.builtin.shell` task.

This should probably be
```.yaml
  - ansible.builtin.set_fact:
      psk: "0s{{ ('Brittany123' | b64encode)[:-5] }}"
```

Reply:
Nice!! I'm doing it now.